_sources/configs/robots.php

```diff
@@ -28,3 +28,10 @@
 }
 
 readfile( 'robots.txt' );
+
+// If the file `extra-robots.txt` is created under the name
+// `/var/www/mediawiki/extra-robots.txt` then its contents get appended to the
+// default `robots.txt` - see MW-312
+if ( file_exists( 'extra-robots.txt' ) && is_readable( 'extra-robots.txt' ) ) {
+	readfile( 'extra-robots.txt' );
```

Comment:
Make sure a new line is added in output between readfile( 'robots.txt' ); and readfile( 'extra-robots.txt' );
in other cases, it can unite the latest rule in robots.txt and the first rule in extra-robots.txt to one line if robots.txt does not end with the new line char.

Reply:
I figured it would be helpful to allow that, but you're probably right - done